extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/remote_inference_client/conversational.py

```diff
@@ -52,10 +53,26 @@ def refine_completion_params(model_settings: dict[Any, Any]) -> dict[str, Any]:
     else:
         # manually set default model because, otherwise this will fail. see comment above.
         # see this thread for more info: https://github.com/huggingface/huggingface_hub/issues/2023
-        completion_data["model"] = "https://api-inference.huggingface.co/models/facebook/blenderbot-400M-distill"
+        completion_data["model"] = (
+            "https://api-inference.huggingface.co/pipeline/conversational/facebook/blenderbot-400M-distill"
+        )
 
     return completion_data
 
+
+def construct_output(response: ConversationalOutput) -> Output:
```

Comment:
Let's add TODO to update this in future to Serverless Endpoint once OpenAI format becomes their standard: https://github.com/huggingface/huggingface_hub/blame/464a34f07f9059ccba9c67ddb7e107cc3184b557/src/huggingface_hub/inference/_types.py#L50